impeller/renderer/backend/gles/texture_gles.h

```diff
@@ -60,13 +60,18 @@ class TextureGLES final : public Texture,
 
   std::optional<GLuint> GetFBO() const { return wrapped_fbo_; }
 
- private:
-  friend class AllocatorMTL;
+  // For non cubemap textures, 0 indicates uninitialized and 1 indicates
+  // initialized. For cubemap textures, each face is initialized separately with
+  // each bit tracking the initialization of the corresponding slice.
+  void MarkSliceInitialized(size_t slice) const;
```

Comment:
This is what I was missing last time. For the GLES cube map textures we need to tracking initialization per slice